OpenSXCE FF port uses GNU ld to link libxul.so
However, our gcc passes some Sun ld-specific options to ld,
so we use wrapper to remove them. Also patch ensures 
that we use GCC 4.9
--- firefox-43.0b4/toolkit/library/autoconflibxul.mk.1	2015-11-23 10:49:55.696324948 +0300
+++ firefox-43.0b4/toolkit/library/autoconflibxul.mk	2015-11-23 10:51:05.252294887 +0300
@@ -26,4 +26,4 @@
 
 //MKSHLIB = /usr/bin/gld $(DSO_PIC_CFLAGS) $(DSO_LDOPTS) -h $(DSO_SONAME) -o $@
 //OS_LDFLAGS = -lpthread	 -z text -z ignore -R,'$$ORIGIN:$$ORIGIN/..' -z lazyload -z combreloc -z muldefs
-MKSHLIB = /usr/bin/env LD_ALTEXEC=/usr/bin/gld PATH=/usr/gcc/4.5/bin:/usr/gcc/4.7/bin:/usr/gcc/4.8/bin:$PATH $(CXX) $(CXXFLAGS) $(DSO_PIC_CFLAGS) $(DSO_LDOPTS) -Wl,-h,$(DSO_SONAME) -o $@
+MKSHLIB = /usr/bin/env LD_ALTEXEC=$(GLD_WRAPPER) PATH=/usr/gcc/4.9/bin:/usr/gcc/4.5/bin:/usr/gcc/4.7/bin:/usr/gcc/4.8/bin:$PATH $(CXX) $(CXXFLAGS) $(DSO_PIC_CFLAGS) $(DSO_LDOPTS) -Wl,-h,$(DSO_SONAME) -o $@
